presto-geospatial/src/main/java/com/facebook/presto/plugin/geospatial/BingTileFunctions.java

```diff
@@ -190,6 +193,136 @@ public static Block bingTilesAround(
         return blockBuilder.build();
     }
 
+    @Description("Given a (longitude, latitude) point, returns the surrounding Bing tiles " +
```

Comment:
Could you add the new function to `presto-docs/src/main/sphinx/functions/geospatial.rst` so that it shows up in the documentation https://prestodb.io/docs/current/functions/geospatial.html ? Functions there are listed in alphabetical order within function groups.

Reply:
Done